docs/getting-started/install.md

```diff
@@ -1,9 +0,0 @@
-Install the latest stable release of the Iter8 CLI as follows.
-
-```shell
-go install github.com/iter8-tools/iter8@v0.16
-```
-
-## Install Iter8 controller
-
---8<-- "docs/tutorials/installiter8controller.md"
```

Comment:
Why are we getting rid of this?

Reply:
We are getting rid of the install of the Iter8 CLI. We are keeping the insetall of the controller. It is documented in another file. No need for this one which does both.